resources/user-agents/browsers/chromium/chromium-28-39.json

```diff
@@ -47,13 +48,13 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (*Linux i686*) applewebkit* (*khtml*like*gecko*) Ubuntu Chromium/#MAJORVER#.*Chrome/*Safari/*",
+          "match": "Mozilla/5.0 (*Linux i686*) applewebkit* (*khtml*like*gecko*) Ubuntu* Chromium/#MAJORVER#.*Chrome/*Safari/*",
```

Comment:
I added the asterisk here so that these patterns will pick up Ubuntu useragents and mark them as Ubuntu, rather than the generic "Linux" (They typically look like `Ubuntu/11.04`).

It may be nice to add more Ubuntu versions and add them to the pattern above, but this should at least catch other versions of Ubuntu that aren't explicitly defined.